File: 프로그래머스/unrated/181934. 조건 문자열/조건 문자열.cs
using System;

public class Solution {
    public int solution(string ineq, string eq, int n, int m) 
    {
        if(ineq == ">")
        {
            if(n < m) return 0;
        }
        else
        {
            if(n > m) return 0;
        }
        
        if(eq == "!")
        {
            if(n == m) return 0;
        }
        
        return 1;
    }
}

File: 프로그래머스/0/181934. 조건 문자열/조건 문자열.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string ineq, string eq, int n, int m) {
    
    if (eq == "=" && n != n)  return 0;
    
    if (ineq == "<")
    {
        if (n > m) return 0;
    }
    else // ">" 
    {
        if (n < m) return 0;
    }
      
    return 1;
}